CHANGELOG.md

```diff
@@ -1,3 +1,214 @@
+GOBBLIN 0.8.0
+-------------
+
+#### Created Date: 08/22/2016
+
+## Highlights
```

Comment:
We should have wiki links for all highlights. You may have to ask people to write one.